### PR TITLE
[UI] Label component and device details page

### PR DESCRIPTION
**Pull request by kuby**

### CODE REVIEW

File: http-gateway/web/src/containers/things/things-list-page.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const Things = () => {
         Header: _(t.name),
         accessor: 'device.n',
         Cell: ({ value, row }) => (
-          <Link to={`/things/${row.values.id}`}>{value}</Link>
+          <Link to={`/things/${row.values?.['device.di']}`}>{value}</Link>
```
Comment (oskarbauer) [inline http-gateway/web/src/containers/things/things-list-page.js]: is "di" correct?  If it is, I think it should be deviceIndentfier or something else like that.   id is clear, di is not

**Review by oskarbauer — DISMISSED** [review]
device.n and devide.di are not clear to me.

**Comment by ondrejtomcik** [comment]
The di represents device id and n device name. Based on OCF spec.

**Review by ondrejtomcik — APPROVED** [review]

File: http-gateway/web/src/containers/things/things-list-page.js (hunk 1 above)
Comment (kuby) [reply http-gateway/web/src/containers/things/things-list-page.js]: That's how it comes from the API (`di` stands for device ID), I can't do much about it :)

**Review by oskarbauer — APPROVED** [review]

**Pull request merged**